core/src/main/java/org/apache/calcite/interpreter/Bindables.java

```diff
@@ -518,28 +522,36 @@ public Node implement(InterpreterImplementor implementor) {
   }
 
   /**
-   * Rule to convert an {@link org.apache.calcite.rel.logical.LogicalUnion}
-   * to a {@link BindableUnion}.
+   * Rule to convert an {@link SetOp} to a {@link BindableUnion}
+   * or {@link BindableIntersect} or {@link BindableMinus}.
    */
-  public static class BindableUnionRule extends ConverterRule {
+  public static class BindableSetOpRule extends ConverterRule {
 
     /**
```

Comment:
Looks like it's a break change. So we'd better deprecate it first.

Reply:
you mean we should implement BindableUnionRule、BindableIntersectRule、BindableMinusRule separately ?

Reply:
Can you add something like this?
```java
@Deprecate // use BindableSetOpRule instead
public static class BindableUnionRule extends BindableSetOpRule {}
```